public/js/main.js

```diff
@@ -1,6 +1,20 @@
 $(document).ready( function() {
 
-  var expectedKeys = ['Text_Input', 'Phone', 'Zip', 'Offer_Type', 'Offer_Detail', 'Need_Type', 'Needs_Detail', 'Language', 'Name', 'Email', 'Housing'];
+  var expectedKeys = [
+    'Text_Input',
+    'Phone',
+    'Zip',
+    'Zip_Home',
+    'Rent',
```

Comment:
We can remove this Rent field as we are not using it currently

Reply:
fine